src/components/esphome-header-menu.ts

```diff
@@ -46,6 +51,7 @@ export class ESPHomeHeaderMenu extends LitElement {
       >
         <mwc-icon-button slot="trigger" icon="more_vert"></mwc-icon-button>
         <mwc-list-item>Update All</mwc-list-item>
+        <mwc-list-item>.Secrets Editor</mwc-list-item>
```

Comment:
We shouldn't add this. There is one way of editing secrets in the frontend and that's the other menu item, regardless of where the files are stored in the backend.